controllers/component_build_controller_pac.go

```diff
@@ -419,6 +443,12 @@ func (r *ComponentBuildReconciler) ensurePaCRepository(ctx context.Context, comp
 				return err
 			}
 			if err := r.Client.Create(ctx, repository); err != nil {
+				if strings.Contains(err.Error(), "repository already exist with url") {
```

Comment:
k8sErrors.IsAlreadyExists  should work?

Reply:
I think `IsAlreadyExists` will not work in this case.

Reply:
I see, ok